src/types/target.ts

```diff
@@ -121,20 +121,20 @@ export class Target implements TargetJson {
   }
   private _streamConfig: StreamConfig | undefined
 
-  get macroFolders(): string[] {
+  get macroFolders(): string[] | undefined {
```

Comment:
I would make a type

Reply:
can you explain a bit?

Reply:
dissmiss my comment - I've been thinking about something else